### PR TITLE
Change velocity test to a square

### PR DESCRIPTION
**Pull request by aftersomemath**

### CODE REVIEW

**Review by asaba96 — COMMENTED** [review]
Can we get rid of the negative zeroes? 😬

File: scripts/velocity_test.py [hunk 1]
```diff
@@ -34,42 +34,34 @@ def velocity_test():
...
-    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=0.3, y_velocity=0.0)
-    # Sends the goal to the action server.
-    client.send_goal(goal)
-    rospy.sleep(5.0)
     client.cancel_goal()
     rospy.logwarn("Translation 1 canceled")
     rospy.sleep(2.0)
 
-    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=-0.0, y_velocity=-0.3)
+    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=-0.0, y_velocity=-0.2)
```
Comment (asaba96) [inline scripts/velocity_test.py]: Since my comment didn’t make it to slack, can we get rid of the negative zeroes 😬

Comment (aftersomemath) [reply scripts/velocity_test.py]: Done

**Review by amiller27 — APPROVED** [review]

**Comment by asaba96** [comment]
I approve looks good to me

**Pull request merged**